<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelTrack</class>
 <widget class="QScrollArea" name="PanelTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>449</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutToolbar">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolbar">
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="labelDirectionScheme">
        <property name="text">
         <string>Scheme</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSolidColor">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QtColorPicker" name="colorPickerSolidColor" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="pushButtonShowClusterMap">
        <property name="text">
         <string>Show Cluster Map</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color code</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBoxDirectionScheme">
        <item>
         <property name="text">
          <string>End Points</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mid Segment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every Segment</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Render</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelDirectionMapping">
        <property name="text">
         <string>Mapping</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>File name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxRenderRep">
        <item>
         <property name="text">
          <string>Lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tubes (Slow!)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBoxDirectionMapping">
        <item>
         <property name="text">
          <string>RAS -&gt; RGB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RAS -&gt; RBG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RAS -&gt; GRB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RAS -&gt; GBR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RAS -&gt; BRG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RAS -&gt; BGR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxColorCode">
        <item>
         <property name="text">
          <string>Directional</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Embedded Color</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOpacity">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>240</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QWidget</extends>
   <header>qtcolorpicker.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
    <signal>currentColorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
